package.json

```diff
@@ -104,6 +106,8 @@
     "eslint-plugin-promise": "^5.1.0",
     "eslint-plugin-standard": "^5.0.0",
     "eslint-plugin-vue": "^7.17.0",
+    "eslint-plugin-vuejs-accessibility": "^1.2.0",
+    "fast-glob": "^3.2.7",
```

Comment:
a newer version of fast-glob is available. also curious as to why this package is added? is it needed by vuejs-accessibility?